cmd/utils/flags.go

```diff
@@ -724,6 +724,12 @@ var (
 		EnvVars:  []string{"BUILDER_PROPOSER_SIGNING_ADDRESS"},
 		Category: flags.BuilderCategory,
 	}
+	BuilderExtraData = &cli.StringFlag{
```

Comment:
this is confusing - does the extra data come from this flag or the payload attributes from the proposer?

Reply:
The extra data should be determined by the builder. so this flag sets the extra data of the payload

Reply:
then why is it in the `BuilderPayloadAttributes` from the proposer?
I think you only need the extra data in the miner.generateParams